### DISCUSSION TITLE
Recognize won't use faces from Contact photos

### DISCUSSION THREAD

**Discussion by teije** [nextcloud/recognize#579] [Q&A] [answered]
First of all I'd like to start this post with a thank you for @marcelklehr and the other contributers of this recognition app. It works amazing + it is free, this is awesome.

My question:
Right at the start of the Recognize App plugin the following statement is made;

> 📷 👪 Recognizes faces from contact photos

So, I have gone and created about 30 contacts and assigned pictures to their contact profile. I assumed that Recoginze would automatically use these faces and assign the names to recoginzed faces where possible. However, it does not do this. When I go to view the faces in the NC Photos or Memories apps I just see a huge list of faces with howmany times these faces are recognized.

How do I make sure Recognize actually uses my contacts and their photos to automatically assign the names?

**Comment by marcelklehr** [comment, marked as answer]
> How do I make sure Recognize actually uses my contacts and their photos to automatically assign the names?

Recognize version 2 used to use the contact images, but in version 3 this functionality was removed for simplicty, when refactoring the app to be more scalable. See #498
